core/templates/dev/head/exploration_editor/exploration_editor.html

```diff
@@ -179,7 +179,7 @@
     <li ng-if="isEditableOutsideTutorialMode()">
       <div dropdown class="btn-group" style="margin-right: 10px; margin-top: 8px;"
            title="<[getSaveButtonTooltip()]>">
-        <button id="tutorialSaveButton" class="btn btn-default oppia-save-draft-button protractor-test-save-changes" ng-class="{'btn-success': isExplorationSaveable()}" ng-click="saveChanges()" ng-disabled="!isExplorationSaveable()">
+        <button id="tutorialSaveButton" class="btn btn-default oppia-save-draft-button protractor-test-save-changes" style="padding-right: 130px;" ng-class="{'btn-success': isExplorationSaveable()}" ng-click="saveChanges()" ng-disabled="!isExplorationSaveable()">
```

Comment:
Could we find a way to not do this styling here? It makes the text very strangely left aligned instead of centered on the button. This is especially obvious when the button says "Save Draft" (the style affects the button then too). Maybe you should take a look at how the styling was done originally. FYI, much of the style may be in oppia.css, and it is good to put style changes in there too.

As another note, the dropdown that says "Discard Changes" should be the same width as this button.